language/llama2-70b/processorca.py

```diff
@@ -208,18 +227,26 @@ def generate(self, export_dir: os.PathLike, n_samples: int = 24576, use_cached:
         sampled_fpath = export_dir / f"open_orca_gpt4_tokenized_llama.sampled_{n_samples}.pkl"
         sampled_df.to_pickle(sampled_fpath)
 
+        # Calibration dataset
+        calib_ds = sampled_df.sample(n=self.calibration_subset_size,
+                                     random_state=12345)
```

Comment:
Shall we make 12345 a macro somewhere?

Reply:
I made it a keyword arg to `generate()`, would that suffice?

Reply:
That should be good